src/Core/Email/SmtpDataConfigurator.php

```diff
@@ -1,101 +0,0 @@
-<?php
-/**
- * 2007-2020 PrestaShop SA and Contributors
- *
- * NOTICE OF LICENSE
- *
- * This source file is subject to the Open Software License (OSL 3.0)
- * that is bundled with this package in the file LICENSE.txt.
- * It is also available through the world-wide-web at this URL:
- * https://opensource.org/licenses/OSL-3.0
- * If you did not receive a copy of the license and are unable to
- * obtain it through the world-wide-web, please send an email
- * to license@prestashop.com so we can send you a copy immediately.
- *
- * DISCLAIMER
- *
- * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
- * versions in the future. If you wish to customize PrestaShop for your
- * needs please refer to https://www.prestashop.com for more information.
- *
- * @author    PrestaShop SA <contact@prestashop.com>
- * @copyright 2007-2020 PrestaShop SA and Contributors
- * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
- * International Registered Trademark & Property of PrestaShop SA
- */
-
-namespace PrestaShop\PrestaShop\Core\Email;
-
-use PrestaShop\PrestaShop\Core\Configuration\DataConfigurationInterface;
-use PrestaShop\PrestaShop\Core\ConfigurationInterface;
-
-/**
- * Class SmtpDataConfigurator is responsible for configuring SMTP data.
- */
-final class SmtpDataConfigurator implements DataConfigurationInterface
```

Comment:
Shouldn't we keep this class, but deprecate it ?

Reply:
You're right, I will do it for all deleted / unused files :+1:

Reply:
Done